mssqlcli/main.py

```diff
@@ -623,13 +559,38 @@ def _handle_server_closed_connection(self):
             show_default=False, type=bool, default=True)
         if reconnect:
             try:
-                reset_connection_and_clients(self.sqltoolsclient,
-                                             self.mssqlcliclient_query_execution)
+                self.reset_connection()
 
                 click.secho('Reconnected!\nTry the command again.', fg='green')
             except Exception as e:
                 click.secho(str(e), err=True, fg='red')
 
+    def reset_connection(self):
+        """

```

Comment:
Maybe just call this reset?

Reply:
If you change this, please update the method name in the logging below.

---
In reply to: [170392798](https://github.com/dbcli/mssql-cli/pull/170#discussion_r170392798) [](ancestors = 170392798)

Reply:
It sounds more intuitive to have reset_connection because that is explicitly resetting, what do you think? 

Having just reset can imply a lot of options to be reset for MssqlCli.

---
In reply to: [170392798](https://github.com/dbcli/mssql-cli/pull/170#discussion_r170392798) [](ancestors = 170392798)